model/attributes.yaml

```diff
@@ -1,187 +1,208 @@
-            "Unknown",
-            "Pass",
-            "Warning",
-            "Fail",
-            "Other",
-          ]
+          [ "OSPS-QA-07.01" ]
         requirement_level: required
-      - id: compliance.audit.id
+      - id: compliance.control.category
         type: string
         stability: development
         brief: >
-          A unique identifier for a specific audit or assessment.
+          Category or family that the security control belongs to.
         examples:
-          ["my_audit_id"]
-        note: |
-          This aligns with a id field of a `gemara` Evaluation or Enforcement Plan. The goal to allow batching
-          and correlation of a set of findings.
+          [ "Access Control", "Quality"]
         requirement_level: recommended
-      - id: compliance.control.id
+      - id: compliance.control.catalog.id
         type: string
         stability: development
         brief: >
-          The unique identifier for the security control.
-        note: |
-          A control is a prescriptive, actionable set of 
-          specifications that strengthens security and compliance posture. This value may also reference
-          a specific control part.
+          Unique identifier for the security control catalog or framework.
         examples:
-          [ "OSPS-QA-07.01" ]
+          [ "OSPS-B", "CCC", "CIS"]
         requirement_level: required
-      - id: compliance.control.remediation.description
-        type: string
+      - id: compliance.control.applicability
+        type: string[]
         stability: development
         brief: >
-          The description of the remediation strategy.
-        examples:
-          [
-            "This is a short description of the remediation strategy for this control.",
-          ]
+          Environments or contexts where this control applies.
+        examples: [ [ "Production", "Staging" ], [ "All Environments" ], [ "Kubernetes", "AWS" ] ]
         requirement_level: opt_in
-      - id: compliance.control.catalog.id
-        type: string
+      - id: compliance.frameworks
+        type: string[]
         stability: development
         brief: >
-          The unique identifier for the security control catalog.
-        examples:
-          [
-            "OSPS-B",
-            "CCC",
-          ]
-        requirement_level: required
+          Regulatory or industry standards being evaluated for compliance.
+        examples: [ [ "NIST-800-53", "ISO-27001" ] ]
+        requirement_level: recommended
       - id: compliance.requirements
         type: string[]
         stability: development
         brief: >
-          The identifiers specific compliance requirements being evaluated.
-        examples:
-          [
-            ["AC-1", "PS-1", "2.1"]
-          ]
+          Compliance requirement identifiers from the frameworks impacted.
+        examples: [ [ "AC-1", "A.9.1.1" ]]
+        requirement_level: recommended
+      - id: compliance.status
```

Comment:
Is this a "expected" status from the policy engine result, correct? I am trying to think about the information life-cycle, such what is pre-defined and what is reported...